net/tor/src/opnsense/mvc/app/models/OPNsense/Tor/Relay.xml

```diff
@@ -24,9 +25,14 @@
     </directory_port>
     <address type="TextField">
       <Required>N</Required>
+      <!-- hostname -->
+      <mask>/^[a-z0-9._-]+$/i</mask>
```

Comment:
underscore is not a valid hostname character

Reply:
will change that - thanks.